azure-devops/azext_devops/dev/pipelines/_format.py

```diff
@@ -208,6 +208,17 @@ def transform_pipeline_runs_table_output(result):
     return table_output
 
 
+def transform_pipeline_or_run_table_output(result):
+    table_output = [_transform_pipeline_or_run_row(result)]
+    return table_output
+
+
+def _transform_pipeline_or_run_row(row):
+    if row.get("buildNumber", None):
```

Comment:
`if row.get("buildNumber", None):`
looks like a hack to detect response type based on presence of a field..
lets add a comment or can we come up with some better way to check type?